### ISSUE TITLE
module.exports['browser'] <-- typo?

### ISSUE THREAD

**Issue by ibc** [ded/bowser#71]
```
!function (name, definition) {
  if (typeof module != 'undefined' && module.exports) module.exports['browser'] = definition()
  else if (typeof define == 'function' && define.amd) define(definition)
  else this[name] = definition()
}('bowser', function () {
```

May be it should be `module.exports['bowser']`?

**Comment by ibc** [comment]
OK, thanks.

**Comment by ded** [comment]
there's no right or wrong answer to this. we'd have to bump it to a major version, which is fine...

**Comment by ibc** [comment]
OK, note that it would be more consistent (same behavior and API for both browser and Node).
Thanks.

**Comment by vvo** [comment]
10 minutes wasted because of this. What do we do?

**Comment by ibc** [comment]
`var bowser = require('bowser')` should export the `bowser` module, instead of an Object with a field called `browser` or `bowser`, so its API would be consistent regardless the library is used as a standalone `<script>` tag or as a AMD/CommonJS module.

**Comment by vvo** [comment]
fixed this @ded

**Comment by ded** [comment]
hey all — this has been fixed in the `1.0.0` version. Thus `require('bowser')` will work the same as it does in Node.js :+1:
